README.md

```diff
@@ -7,26 +7,32 @@ NOTE: This project is currently in the Pilot phase and is covered by Segment's [
 to try out this new library. Please provide feedback via Github issues/PRs, and feel free to submit pull requests.  This library will eventually 
 supplant our `analytics-android` library.
 
+NOTE: If you use pure Java codebase, please refer to [Java Compatibility](JAVA_COMPAT.md) for sample usages.
+
 ## Table of Contents
-- [Installation](#installation)
-- [Usage](#usage)
+- [Analytics-Kotlin](#analytics-kotlin)
+  - [Table of Contents](#table-of-contents)
+  - [Installation](#installation)
+    - [Permissions](#permissions)
+  - [Usage](#usage)
     - [Setting up the client](#setting-up-the-client)
     - [Client Options](#client-options)
-- [Client Methods](#client-methods)
-    - track
-    - identify
-    - screen
-    - group
-    - add
-    - find
-    - remove
-    - flush
-- [Plugin Architecture](#plugin-architecture)
+  - [Client Methods](#client-methods)
+    - [track](#track)
+    - [identify](#identify)
+    - [screen](#screen)
+    - [group](#group)
+    - [alias](#alias)
+    - [add](#add)
+    - [find](#find)
+    - [remove](#remove)
+    - [flush](#flush)
+  - [Plugin Architecture](#plugin-architecture)
     - [Fundamentals](#fundamentals)
-    - [Advanced Concepts](#advanced-concepts)
-- [Contributing](#contributing)
-- [Code of Conduct](#code-of-conduct)
-- [License](#license)
+    - [Advanced concepts](#advanced-concepts)
+  - [Contributing](#contributing)
```

Comment:
whats the reason for this change?

Reply:
I used a vcs plugin to update the table of content. it changes the content automatically. it's mainly for adding `alias` to table of content